tests/plugins/pastefromword/generated/_helpers/createTestCase.js

```diff
@@ -1,5 +1,4 @@
-/* global assertWordFilter,Q */
-/* bender-include: _helpers/pfwTools.js */
+/* global assertWordFilter,Q,console */
 /* exported createTestCase */
 
 // @param {boolean} [compareRawData=false] If `true` test case will assert against raw paste's `data.dataValue` rather than
```

Comment:
If we're adding docs in other places, then we can also add missing docs for `createTestCase` as only one parameter is described.